src/VocabularyManager.Api/Controllers/WordsController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using VocabularyManager.Api.ActionFilters;
+using VocabularyManager.Core.Entities;
+using VocabularyManager.Core.Interfaces;
+
+namespace VocabularyManager.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class WordsController : ControllerBase
+    {
+        private readonly IWordStoreManager _wordStoreManager;
+        private readonly IVocabularyStoreManager _vocabularyStoreManager;
+        public WordsController(
+            IWordStoreManager wordStoreManager,
+            IVocabularyStoreManager vocabularyStoreManager)
+        {
+            _wordStoreManager = wordStoreManager;  
+            _vocabularyStoreManager = vocabularyStoreManager;
+        }
+        [HttpPost]
+        [ServiceFilter(typeof(WordValidationFilter))]
+        public async Task<IActionResult> AddWord(
+            [FromQuery] int vocabularyId,
+            [FromBody] Word word)
+        {
+            int addedWordId = await _vocabularyStoreManager.AddWord(word, vocabularyId);
+            return Ok(addedWordId);
+        }
+        [HttpPost("addrange")]
```

Comment:
what's difference between adding one item and adding multiple items? For me, It's the same ;) 

Think about it how you can change your code